source/utils/ResourceManager.cpp

```diff
@@ -133,6 +134,8 @@ void ResourceManager::setupDataPath(boost::program_options::variables_map& optio
         {
             mGameDataPath.append("/");
         }
+
+        mGameDataPath = Ogre::FileSystemLayer::resolveBundlePath(mGameDataPath);
```

Comment:
this can also resolve the mac bundle path, but I cannot test that and I wanted to keep changes minimal